### ISSUE TITLE
Error after installing via Pod

### ISSUE THREAD

**Issue by Dirk23** [amosavian/FileProvider#160]
Hi,

I added FileProvider to my App, to have an alternative to Dropbox for Filesync. After I added `pod "FilesProvider"` to my Podfile I ran `pod update` and everything seems ok:
```Update all pods
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Installing FilesProvider (0.26.0)
Generating Pods project
Integrating client project
Pod installation complete! There are 3 dependencies from the Podfile and 4 total pods installed.
```
When I now open the Workspace of my App, Xcode tells me that there are some Errors:
![Bildschirmfoto 2020-04-22 um 09 28 48](https://user-images.githubusercontent.com/4200682/79953147-b9b96400-847b-11ea-9619-79ad82074b81.png)

![Bildschirmfoto 2020-04-22 um 09 29 28](https://user-images.githubusercontent.com/4200682/79953198-ca69da00-847b-11ea-99f5-2f5b3168fb84.png)

![Bildschirmfoto 2020-04-22 um 09 29 55](https://user-images.githubusercontent.com/4200682/79953245-d9508c80-847b-11ea-91e6-e4e5fe68e090.png)

And I don't understand what the Problem is. Is it maybe due to Swift 5?

**Comment by alopez-hi** [comment]
I've successfully add this pod to my project. My `podfile` looks like:
```
# Uncomment the next line to define a global platform for your project
platform :ios, '13.0'

target 'myapp' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for myapp
  pod 'FilesProvider', '~> 0.26'
end
```

The installed version is: `FilesProvider (0.26.0)`

My CocoaPods gem is: `pod --version` returns `1.9.2`

Maybe removing the pod, cleaning project and re-installing. Sometimes works for me when I've some trouble with a pod.

**Comment by Dirk23** [comment]
@alopez-hi No, I think it is a problem with Swift5. It seems that the `case .DATA_ERROR:` creates some kind of custom Error in Xcode. When I change the variable Name from `DATA_ERROR` to `DATA_ERRORS` the Error Message disappeares.
